mne/viz/evoked.py

```diff
@@ -1110,24 +1110,24 @@ def plot_evoked_joint(evoked, times="peaks", title='', picks=None,
     title : str | None
         The title. If `None`, suppress printing channel type. Defaults to ''.
     picks : array-like of int | None
-        The indices of channels to plot. If None show all. Defaults to None.
+        The indices of channels to plot. If `None` show all. Defaults to None.
     exclude : None | list of str | 'bads'
         Channels names to exclude from being shown. If 'bads', the
         bad channels are excluded. Defaults to None.
     show : bool
         Show figure if True. Defaults to True.
     ts_args : None | dict
         A dict of `kwargs` that are forwarded to `evoked.plot` to
-        style the butterfly plot. `axes` and `show` are ignored.
-        If `spatial_colors` is not in this dict, `spatial_colors=True`,
-        and (if it is not in the dict) `zorder='std'` will be passed.
-        Defaults to ``None``.
+        style the butterfly plot. If `spatial_colors` is not in this dict,
+        `spatial_colors=True`, and (if it is not in the dict) `zorder='std'`
+        will be passed. `axes`, `show`, `exclude` are illegal.
+        If `None`, no customizable arguments will be passed.
+        Defaults to `None`.
     topomap_args : None | dict
-        A dict of `kwargs` that are forwarded to `evoked.plot_topomap`
-        to style the topomaps. `axes` and `show` are ignored. If `times`
-        is not in this dict, automatic peak detection is used. Beyond that,
-        if ``None`, no customizable arguments will be passed.
-        Defaults to ``None``.
+        A dict of `kwargs` that are forwarded to `evoked.plot_topomap` to
+        style the topomaps. `axes`, `show`, `times`, `colorbar` are illegal.
+        If `None`, no customizable arguments will be passed.
+        Defaults to `None`.
```

Comment:
I'm not sure about the styling, but I probably wouldn't use backticks with None and use double backticks with the others.